### ISSUE TITLE
The keyboard focus is traversing the outer-boundary of feedback window, but the keyboard focus on that window is not visible, which lead to focus loss for the feedback window in the Xamarin Inspector Application.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#287]
### Pre-requisites:

Launch the Visual Studio for Mac and Sign-in with Enterprise account.
Create projects:
1. Go to Visual Studio-> File-> New Solution. -> Multiplatform-> App-> Xamarin.Forms-> Blank Forms App.
2. Hit on Next button to create the project.
3. Hit on Debug(Play)/Build button to build/deploy the solution/project.
4. As per the choice/requirements, Change the Device platform like iOS/ Android and iPhone X iOS 11.3 etc.
5. Hit on Inspector icon button (i.e., in the top toolbar buttons of the VS) to inspect the project.
6. On hitting on inspector button, Xamarin Inspector is Launched.

### Repro-Steps:

1. Follow the Pre-requisites to Launch the Xamarin Inspector from Visual Studio.
2. Navigate to Xamarin Inspector-> preferences->Hit On Enter key, which opens the General tab by default.
3. Navigate to Feedback tab-> Hit on Enter to open Feedback tab window.
4. Press on the Tab key to traverse all the interactive elements of this opened feedback tab.
5. Observe the keyboard focus next to Feedback tab item.

### Expected Behavior
The keyboard focus is traversing the outer-boundary of the feedback window, so keyboard focus should be visible on this window, otherwise it would be also better if keyboard traversing is removed from outer boundary.
### Actual Behavior
The keyboard focus is traversing the outer-boundary of feedback window, but that keyboard focus on that window is not visible, which lead to focus loss for the feedback window.
### User Impact
The keyboard user might be confused due to apparent focus loss. 

![kb3_feedbackouterboundary_focusnotvisible](https://user-images.githubusercontent.com/22956130/39982766-02881b0a-5773-11e8-8124-3ab8393de3af.png)

### Environment
Visual Studio for Mac Enterprise [Version: 7.5 Preview (7.5 build 1222)]
Xamarin Inspector [Version: 1.4 (master/b3f92f9)]

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Inspector-MAC
Wipro
MAS2.4.7

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1830
